packages/frontend-shared/README.md

```diff
@@ -18,26 +22,143 @@ For the best development experience, you will want to use VS Code with the [Vola
+
+* Most of our icons are duo-tone
+* They must be styled in both dark and light contexts (e.g. on a dark menu bar vs on a light background).
+* Since they're duotone, you want to target the specific strokes and fills of the SVGs to color them.
+* All of the niceties of our utility classes should work (e.g. `group-hover` or `group-focus`).
+
+### Usage
+#### Importing Icons
+1. `import MyIcon from '~icons/cy/path-to-icon_x16'`
+2. Automatic template discovery
+```jsx
+/* This just works. No imports necessary */
+<i-cy-path-to-icon_x16 />
+```
+
+#### Styling Icons
+```jsx
+/* This renders a book icon from `./src/assets/book_x16.svg`
+ * and makes it pink and purple
+ */
+
+<i-cy-book_x16 class="
+  icon-dark-pink-300
+  icon-dark-purple-50
+  hover:icon-dark-pink-500
+  hover:icon-light-purple-300
+" />
+```
+
+### Implementation: Custom classes w/ a WindiCSS plugin
+To support selecting specific paths while keeping Tailwind's incredibly helpful interaction helpers (e.g. `group-hover` or `group-focus`), we use a WindiCSS plugin. Windi configuration lives in the [windi.config.ts](windi.config.ts) file in this package.
+#### Adding new icons
+
+To add an new icon: 
+
+1. Export the icon from Figma (all icons should come from the design system in Figma) as an SVG. 
+1. Add the SVG to the [icons folder](./src/assets/icons). 
+1. Name the file following the existing convention in there: `icon-name_x[size in pixels]`, e.g. `arrow-down_x16.svg`.
+1. Manually edit the SVG file to add classes `icon-dark` and `icon-light` to the dark and light internal elements, and save the file. If an icon path doesn't define a class, nothing bad will happen, it just won't get targeted by any styling.
+1. Finally, you don't need to expose anything. `./src/assets/icons` is automatically watched and loaded 😮
+
+Now the icon is ready to be used in Vue SFC templates  
+### Accessibility
+
+We consider accessibility a core part of front-end code quality. When possible, components should be built out using standard semantic HTML elements. If there are no plain HTML solutions for a particular interaction, we can reach for a library (like HeadlessUI above) that implements known patterns in an accessible way. In rare cases we will augment our HTML with ARIA roles. Tests should use the accessible name or label for interactive elements as described in the [testing guide](../../guides/testing-strategy-and-styleguide.md).
+#### The Accessibility Tree
+
+The [Accessibility Tree](https://developers.google.com/web/fundamentals/accessibility/semantics-builtin/the-accessibility-tree) available in your browser's dev tools will show how the nature, structure, and labelling of the elements in the DOM is presented to assistive technology. We can use this to explore the accessibility of our components, especially when creating or reviewing more complex UI interactions, to make sure the content that appears there makes sense.
+### GraphQL in Vue Components
```

Comment:
Let's move this up to be a child of
```
## Use of graphql by shared components is limited to `src/gql-components`
```
(or make this the ## header and that one a child of it).